plugins/utils/wilder-nvim.nix

```diff
@@ -0,0 +1,80 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.wilder-nvim;
+  helpers = import ../helpers.nix {inherit lib;};
+
+  boolToInt = value:
+    if value == null
+    then null
+    else if value
+    then "1"
+    else "0";
+in {
+  options.plugins.wilder-nvim =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "wilder-nvim";
+
+      package = helpers.mkPackageOption "wilder-nvim" pkgs.vimPlugins.wilder-nvim;
+
+      modes = helpers.defaultNullOpts.mkNullable (types.listOf types.str) "['/', '?']" ''
```

Comment:
The default value should be the nix version (as it is shown in the documentation and is easier to copy paste), this means `["/" "?"]`.

Reply:
Right always get mixed up how strings work in Nix, it should be fixed now.